src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Runtime/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services Language LUIS")]
 [assembly: AssemblyDescription("Provides API functions for consuming the Microsoft Azure Cognitive Services LUIS Runtime API.")]
 
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.1.0.0")]
+[assembly: AssemblyVersion("2.8.0.0")]
```

Comment:
`AssemblyVersion` remains the same, only `AssemblyFileVersion` must be bumped for a non-major version bumo